cmd/opampsupervisor/supervisor/config/config.go

```diff
@@ -185,6 +185,7 @@ type Agent struct {
 	Description             AgentDescription  `mapstructure:"description"`
 	ConfigApplyTimeout      time.Duration     `mapstructure:"config_apply_timeout"`
 	BootstrapTimeout        time.Duration     `mapstructure:"bootstrap_timeout"`
+	HealthCheckHost         string            `mapstructure:"health_check_host"`
 	HealthCheckPort         int               `mapstructure:"health_check_port"`
```

Comment:
I think we should probably just remove this property entirely, but if we keep it, can we make it an endpoint instead of just a port?